qtcreator.pri

```diff
@@ -169,10 +169,12 @@ gcc:!clang: QMAKE_CXXFLAGS += -Wno-noexcept-type
 
 RELATIVE_PLUGIN_PATH = $$relative_path($$IDE_PLUGIN_PATH, $$IDE_BIN_PATH)
 RELATIVE_LIBEXEC_PATH = $$relative_path($$IDE_LIBEXEC_PATH, $$IDE_BIN_PATH)
+RELATIVE_REVERSE_LIBEXEC_PATH = $$relative_path($$IDE_BIN_PATH, $$IDE_LIBEXEC_PATH)
 RELATIVE_DATA_PATH = $$relative_path($$IDE_DATA_PATH, $$IDE_BIN_PATH)
 RELATIVE_DOC_PATH = $$relative_path($$IDE_DOC_PATH, $$IDE_BIN_PATH)
 DEFINES += $$shell_quote(RELATIVE_PLUGIN_PATH=\"$$RELATIVE_PLUGIN_PATH\")
 DEFINES += $$shell_quote(RELATIVE_LIBEXEC_PATH=\"$$RELATIVE_LIBEXEC_PATH\")
+DEFINES += $$shell_quote(RELATIVE_REVERSE_LIBEXEC_PATH=\"$$RELATIVE_REVERSE_LIBEXEC_PATH\")
```

Comment:
These are only used in merssh. Please consider having the definitions in merssh.pro only.

Reply:
right, that would be better.